docker/backup.postgres.plugin

```diff
@@ -58,6 +58,7 @@ function onRestoreDatabase(){
     _portArg=${_port:+"-p ${_port}"}
     _username=$(getUsername ${_databaseSpec})
     _password=$(getPassword ${_databaseSpec})
+    _schema='public'
```

Comment:
Why is the schema being hard coded here when it is user definable through `TABLE_SCHEMA` which defaults to `public`?